lib/service_check.pm

```diff
@@ -263,10 +264,16 @@ sub install_services {
             }
         };
         if ($@) {
+            $service->{$s}->{before_migration} = 0;
```

Comment:
Could you use a function to do this part? It seems the only difference is 'before' or 'after' here.

Reply:
It's just 2 line of code, if we use function we'd have hash reference parameter and then decode it in the funciton. which will more than 2 loc.
thanks